src/main/java/nl/mpcjanssen/simpletask/util/Util.kt

```diff
@@ -474,48 +475,77 @@ fun readAsset(assets: AssetManager, name: String): String {
     return buf.toString()
 }
 
-fun getRelativeThresholdDate(task: Task, ctx: Context): String? {
-    val date = task.thresholdDate
-    if (date != null) {
-        date.toDateTime()?.let {
-            return "T: " + RelativeDate.getRelativeDate(ctx, it)
-        }
-    }
-    return null
+fun getRelativeThresholdDate(task: Task, app: TodoApplication): String? {
+    val date = task.thresholdDate ?: return null
+    return getRelativeDate(app, "T: ", date).toString()
 }
```

Comment:
Just making this a little more compact and readable. Also passing date in as a string instead of a DateTime. Fewer indentations are better, right?